build-static.sh

```diff
@@ -137,7 +137,14 @@ else
 	${spcCommand} build --debug --enable-zts --build-embed ${extraOpts} "${PHP_EXTENSIONS}" --with-libs="${PHP_EXTENSION_LIBS}"
 fi
 
-if ! type "xcaddy" >/dev/null 2>&1; then
+if ! type "go" >/dev/null 2>&1; then
+	echo "The \"go\" command must be installed. We strongly recommended installed Homebrew for automatically install Golang"
+     	exit 1
+fi
+
+XCADDY_COMMAND="$(go env GOPATH)/bin/xcaddy"
```

Comment:
We should check if `xcaddy` is already in the path, and only if it isn't install it using `go install` and use the absolute path.
This would allow users to use existing local installations (for instance, when using the Debian packages provided by Caddy, or custom versions with ad hoc patches).